Templates/SaaSFileTrigger-Batch/run.bat

```diff
@@ -0,0 +1,5 @@
+@echo off
+
+type %input% >> %output%
```

Comment:
might consider using `copy %input% %output%` like the BlobTrigger-Batch sample does.